### PR TITLE
Servlet extension improvements

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: docs/modules/ROOT/pages/reference/extensions/servlet.adoc [hunk 1]
```diff
@@ -83,6 +193,30 @@ A servletName as it would be defined in a `web.xml` file or in the `jakarta.serv
...
+
+|icon:lock[title=Fixed at build time] [[quarkus.camel.servlet.force-await]]`link:#quarkus.camel.servlet.force-await[quarkus.camel.servlet.force-await]`
+
+When set to `true` used in conjunction with `quarkus.camel.servlet.async = true`, this will force route processing to run synchronously.
+| `boolean`
+| `false`
+
+|icon:lock[title=Fixed at build time] [[quarkus.camel.servlet.executor-ref]]`link:#quarkus.camel.servlet.executor-ref[quarkus.camel.servlet.executor-ref]`
+
+The name of a bean of to configure an optional custom thread pool for handling Camel Servlet processing.
```
Comment (aldettinger) [inline docs/modules/ROOT/pages/reference/extensions/servlet.adoc:216]: Maybe "The name of a bean to configure" ?

**Review by aldettinger — APPROVED** [review]
LGTM The deployment test is great value with minimal code. Also comments are really helpful to grasp the intent :+1:

File: docs/modules/ROOT/pages/reference/extensions/servlet.adoc (hunk 1 above)
Comment (jamesnetherton) [reply docs/modules/ROOT/pages/reference/extensions/servlet.adoc:216]: Yep - thanks for catching it. I'll fix it.

Comment (jamesnetherton) [reply docs/modules/ROOT/pages/reference/extensions/servlet.adoc]: I've updated it now.

**Pull request merged**